### PR TITLE
Update FF field names to match #10670

### PR DESCRIPTION
**Pull request by ctzsnooze**
Update Blackbox Log viewer field names to reflect the debug changes in [#10670](https://github.com/betaflight/betaflight/pull/10670)

### CODE REVIEW

File: js/flightlog_fields_presenter.js [hunk 1]
```diff
@@ -411,10 +411,10 @@ function FlightLogFieldPresenter() {
...
             'FF_INTERPOLATED' : {
                             'debug[all]':'FF Interpolated', 
-                            'debug[0]':'Setpoint Delta Impl [Roll]',
-                            'debug[1]':'Boost Amount',
-                            'debug[2]':'Boost Amount Clip [Roll]',
-                            'debug[3]':'Clip',
+                            'debug[0]':'Raw FF Derivative [Roll]',
+                            'debug[1]':'Cleaned FF Derivative ',
+                            'debug[2]':'Cleaned Boost Amount [Roll]',
+                            'debug[3]':'Duplicate Marker',
```
Comment (McGiverGim) [inline js/flightlog_fields_presenter.js:417]: Here we have the habitual nightmare of the blackbox and configurator... we need to maintain backward compatibility :(

I don't remember to have done this before for the debug names, so I can't point to a sample at this moment... but if the "old" name is correct for old logs, then we can't simply rename them.

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: js/flightlog_fields_presenter.js (hunk 1 above)
Comment (ctzsnooze) [reply js/flightlog_fields_presenter.js:417]: I see.  

The current names aren't quite right in a lot of debug cases.  In this case the existing names aren't right for 4.2 either :-)

If its tough to provide a compatibility solution for the debugs, could we make 'releases' that are correct for a given version of the software, in terms of debug names and waveform scaling ?

Comment (McGiverGim) [reply js/flightlog_fields_presenter.js:417]: I doubt. A lot of people have different quads some of them in different versions, and expect all to be working... @mikeller what do you think in this case?
I cant try to look at the code this week to see if we can make this conditional depending on the version.

**Comment by ctzsnooze** [comment]
I've tried a number of things to get blackbox to respond to a change in the names of fields when the debug name hasn't changed.
The easiest way is to add a new debug mode in the firmware, but we shouldn't need to do that.
If someone can figure how to version control those fields. there are quite a few that need updating.  I'll do the work if someone can explain how to do it.

**Comment by ctzsnooze** [comment]
Can be closed since #514 provided version control for debug names and included fixing the feedforward names.  Thanks @McGiverGim

**Pull request closed without merging**